src/main/java/com/github/sylvainlaurent/maven/swaggervalidator/semantic/validator/ValidationContext.java

```diff
@@ -66,4 +72,26 @@ private void setDefinitions(Map<String, Model> definitions) {
     public Map<String, SecuritySchemeDefinition> getSecurityDefinitions() {
         return swagger.getSecurityDefinitions();
     }
+
+    @SuppressWarnings("unchecked")
+    private Set<String> getDefinedMimeTypes() {
+        Set<String> set;
+        try {
```

Comment:
I think this code could be encapsulated in a collaborator class (e.g. MediaType)

Reply:
done